chainer/gradient_check.py

```diff
@@ -138,20 +143,24 @@ def check_backward(func, x_data, y_grad, params=(),
     :func:`numerical_grad` to calculate numerically the gradients and compares
     the types of gradients with :func:`chainer.testing.assert_allclose`.
 
-    To reduce computational time, it uses a function
-    :math:`g: \\mathbb{R} \\rightarrow \\mathbb{R}^n` defined as
-    :math:`g(\\alpha) = f(\\alpha x)`, where :math:`\\alpha \\in \\mathbb{R}`
-    and :math:`f` is a function which actually
-    you want to test.
+    To reduce computational time, it uses directional derivative along a
+    random vector. A function
+    :math:`g: \\mathbb{R} \\rightarrow \\mathbb{R}^n` is defined as
+    :math:`g(\\delta) = f(x + \\delta r)`, where
+    :math:`\\delta \\in \\mathbb{R}`, :math:`r \\in \\mathbb{R}^n`
+    is a random vector
+    and :math:`f` is a function which actually you want to test.
```

Comment:
`... a function which you actually want ...` (or remove `actually`, completely, I think this latter is cleaner).